packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -2092,3 +2092,5 @@ Policy Rule conditions aren't supported for this policy.
 | `profileAttributes` | A list of attributes to prompt the user during registration or progressive profiling. Where defined on the User schema, these attributes are persisted in the User profile. Non-schema attributes may also be added, which aren't persisted to the User's profile, but are included in requests to the Registration Inline Hook. A maximum of 10 Profile properties is supported.                                                         | Array | Required | N/A                                                                                                                                                                                                                        |
 | `targetGroupIds`             | (Optional, max 1 entry) The `id` of a Group that this User should be added to                                                     | Array   | No | N/A                                                                                                                                                                                                                         |
 | `unknownUserAction`          | Which action should be taken if this User is new (Valid values: `DENY`, `REGISTER`)                                               | String  | YES | N/A                                                                                                                                                                                                                        |
+
+> **Note:** The Profile Enrollment Action object can't be modified to set the `access` property to `DENY` after the policy is created.
```

Comment:
Do we want to call this note out in the table above? If that's not the common practice, then 🚢  from me

Reply:
We typically don't add a NOTE in the property description table, but this info could be added as part of the property description. 
I decided to just make it as a note because Hanxing says the behaviour of the `access` property during creation could change in the future. I can add the info to the table if that makes sense to you.